citesphere/src/main/java/edu/asu/diging/citesphere/core/service/IGroupManager.java

```diff
@@ -6,5 +6,6 @@
 public interface IGroupManager {
 
     ICitationGroup getGroup(IUser user, String groupId);
-
+    
+    void deleteGroup(IUser user, String groupId);
```

Comment:
needs javadoc; I would also rename this to something like deleteLocalGroupCopy or something to make clear it doesn't complete delete the group.